### PR TITLE
Transform exercises in Stack projects - Phase 4 - Migrating all exercises - Part 3

### PR DESCRIPTION
**Pull request by rbasso**
###### Phase 4 - Migrating all exercises - Part 3

Migrate to stack projects:

- accumulate
- anagram
- beer-song
- bob
- etl
- gigasecond
- grade-school
- grains
- hamming
- leap
- list-ops
- meetup
- nucleotide-count
- phone-number
- rna-transcription
- robot-name
- simple-linked-list
- space-age
- strain
- sublist
- sum-of-multiples
- word-count

Add incomplete stub solutions:

- beer-song
- grade-school
- leap
- robot-name
- space-age
- sublist
- sum-of-multiples
- word-count

Add complete stub solutions:

- accumulate
- anagram
- bob
- gigasecond
- meetup
- nucleotide-count
- phone-number
- simple-linked-list
- strain

Fix package.yaml comments in all exercises:

- Change the text and move it before the library packages.

### CODE REVIEW

File: exercises/simple-linked-list/src/LinkedList.hs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    , fromList
+    , isNil
+    , new
+    , next
+    , nil
+    , reverseLinkedList
+    , toList
+    ) where
+
+-- The task is to create the data type `LinkedList`
```
Comment (petertseng) [inline exercises/simple-linked-list/src/LinkedList.hs]: hmm, if the task is to create the list, do you intend to include the `data LinkedList a` in the stub?

File: exercises/space-age/src/SpaceAge.hs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module SpaceAge (Planet(..), ageOn) where
+
+-- The task is to create the data type `Planet`
+-- and implement the functions `ageOn`.
```
Comment (petertseng) [inline exercises/space-age/src/SpaceAge.hs]: probably just say "function" here rather than "functions"

**Comment by petertseng** [comment]
Thanks for doing the hard work here, after which we can :tada: 

Just small comment I think

File: exercises/simple-linked-list/src/LinkedList.hs (hunk 1 above)
Comment (rbasso) [inline exercises/simple-linked-list/src/LinkedList.hs]: Ops...gonna fix that as soon as possible.
Thanks. :-)

**Comment by rbasso** [comment]
Thank you for reviewing all this, @petertseng! I know it's a lot of boring boilerplate code...

When do you think it's a good time to merge this and the other PRs ? (#201, #202, #203, #204)

**Comment by petertseng** [comment]
I'll say that I wouldn't fault a decision to merge this PR now. I've already fetched exercises converted in earlier phases without problem. The other possiblity is that if you want to wait for more students to report problems, but I find it unlikely there will be any, as exercism fetching in general (both with or without directories) has been well tested I'm sure

**Pull request merged**